Neos.Flow/Classes/Session/Data/SessionIdentifier.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+namespace Neos\Flow\Session\Data;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Utility\Algorithms;
+
+/**
+ * @Flow\Proxy(false)
+ * @internal
+ */
+class SessionIdentifier
+{
+    private function __construct(
+        public /*readonly*/ $value
```

Comment:
can be _real_ readonly now with Flow 9

Reply:
You are right, i forgot about that. Will go over again and also add proper argument and return types to the whole session classes.

Reply:
is adjusted